api/db-cdr/changelog-data/README-BigData.txt

```diff
@@ -0,0 +1,28 @@
+# Liquibase does not like big .csv imports. So for now, until we build the permanent data copy solutions,
+# to fill the cdr.concept and cdr.concept_relationship tables  for data browser follow these steps :
+# Note , you don’t need do this if you are not running the data browser in the ui or using the api
+
+1. download  concept.csv and concept_relationship.csv to your local computer
```

Comment:
Can we add a TODO to script this?

We will also need scripts or directions to load the data into our Cloud SQL environments. (Has this already been done in test?)

Reply:
OK, clarification: there's no such thing as "local cloud SQL". There's just local MySQL. 

Test = Cloud SQL, and we need to have scripts which import this data into it. If you want to do it in a follow-up that's fine, but we should do it soon, as it won't work in test until that's done.